src/message/messageActionSheet.js

```diff
@@ -436,12 +413,18 @@ export const showTopicActionSheet = ({
     unreadStreams: UnreadStreamsState,
     ...
   }>,
-  stream: string,
+  streamId: number,
   topic: string,
 |}): void => {
+  const sub = backgroundData.subscriptions.find(x => x.stream_id === streamId);
+  if (!sub) {
+    logging.error('Stream id does not exist.');
```

Comment:
Is this quite the right error message? We've got a stream ID; it's the subscription that we haven't found, right?

Also, do we expect the `if (!sub)` condition to ever happen, and why or why not?

If we never expect it to happen, let's use [`invariant`](https://github.com/zulip/zulip-mobile/blob/master/docs/style.md#invariant-assertions) instead (here and in similar places in this commit); the number of places we've used that is small but growing. 🙂

Reply:
Using invariant in the new revision. The reason is explained at https://github.com/zulip/zulip-mobile/pull/4635#discussion_r632862010

Also I changed the error message in `invariant`.